pandas/io/parsers.py

```diff
@@ -3494,15 +3495,6 @@ def _get_col_names(colspec, columns):
     return colnames
 
 
-def _concat_date_cols(date_cols):
-    if len(date_cols) == 1:
```

Comment:
how much of this perf improvement is due to the changes in does_look_like_datetime vs the changes in concat_date_cols?